panic: runtime error: index out of range

Got this error:

````
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/rebuy-de/aws-nuke/resources.(*CloudFormationStack).Remove(0xc0032fd1a0, 0x3167a01, 0x7)
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/resources/cloudformation-stack.go:52 +0x467
github.com/rebuy-de/aws-nuke/cmd.(*Nuke).HandleRemove(0xc000524000, 0xc003365b30)
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/cmd/nuke.go:247 +0x37
github.com/rebuy-de/aws-nuke/cmd.(*Nuke).HandleQueue(0xc000524000)
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/cmd/nuke.go:226 +0x197
github.com/rebuy-de/aws-nuke/cmd.(*Nuke).Run(0xc000524000, 0xb, 0xc00017c070)
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/cmd/nuke.go:90 +0x5d5
github.com/rebuy-de/aws-nuke/cmd.NewRootCommand.func2(0xc00017ec80, 0xc000096820, 0x0, 0x5, 0x0, 0x0)
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/cmd/root.go:62 +0x2da
github.com/spf13/cobra.(*Command).execute(0xc00017ec80, 0xc00001e1f0, 0x5, 0x5, 0xc00017ec80, 0xc00001e1f0)
        /Users/rowan/go/src/github.com/spf13/cobra/command.go:753 +0x473
github.com/spf13/cobra.(*Command).ExecuteC(0xc00017ec80, 0xc000024ef8, 0xc000096780, 0xc0000967d0)
        /Users/rowan/go/src/github.com/spf13/cobra/command.go:843 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xc00017ec80, 0xc00019ff88, 0xc000078058)
        /Users/rowan/go/src/github.com/spf13/cobra/command.go:791 +0x2b
main.main()
        /Users/rowan/go/src/github.com/rebuy-de/aws-nuke/main.go:23 +0x27
````

I'm guessing it's just because I have a silly number of resources?

Not sure if this is worth fixing, feel free to close if not.

--- a/resources/cloudformation-stack.go
+++ b/resources/cloudformation-stack.go
@@ -49,10 +49,10 @@ func (cfs *CloudFormationStack) Remove() error {
 	o, err := cfs.svc.DescribeStacks(&cloudformation.DescribeStacksInput{
 		StackName: cfs.stack.StackName,
 	})
-	stack := o.Stacks[0]
 	if err != nil {
 		return err
 	}
+	stack := o.Stacks[0]
 
 	if *stack.StackStatus != cloudformation.StackStatusDeleteFailed {
 		cfs.svc.DeleteStack(&cloudformation.DeleteStackInput{

